Library/Homebrew/keg.rb

```diff
@@ -105,7 +105,13 @@ def self.find_some_installed_dependents(kegs)
     #
     # This happens after the initial dependency check because it's sloooow.
     remaining_formulae = Formula.installed.select do |f|
-      f.installed_kegs.any? { |k| Tab.for_keg(k).runtime_dependencies.nil? }
+      installed_kegs = f.installed_kegs
+
+      # All installed kegs are going to be removed anyway,
```

Comment:
Sure. The purpose of `Keg.find_some_installed_dependents` is to take some kegs and see if there are any kegs outside of that array that depend on those kegs. (That's why it takes an array.) This is what it [does](https://github.com/Homebrew/brew/blob/c29a458dc15efc92e3649155d0ed44bf982cdc15/Library/Homebrew/keg.rb#L95) currently when the install receipts are reliable.

But, in the fallback, I forgot to remove the passed kegs from the list of dependents, so now I've fixed that.